Mathlib/Data/List/Nodup.lean

```diff
@@ -5,6 +5,7 @@ Authors: Mario Carneiro, Kenny Lau
 -/
 import Mathlib.Data.List.Forall2
 import Mathlib.Data.Set.Pairwise.Basic
+import Mathlib.Data.List.TakeDrop
```

Comment:
We can get rid of the import here since the `simp` on line 244 doesn't actually need `List.drop_length_sub_one`. Hopefully that pushes the import of `TakeDrop` down even more!